### ISSUE TITLE
Icons show up as squares with text inside[BUG]

### ISSUE THREAD

**Issue by Oddiki** [darkreader/darkreader#4736]
It's something that's happened fairly newly, about a month or so. It seems to only happen on some site, can't remember which other sites I've seen it on. But I know it happens on Myanimelist.net, since I frequently visit this site to update my anime viewing.

With Dark mode
https://imgur.com/MigPzrK
Without Dark mode
https://imgur.com/aNaLhvg

Firefox 84.0.2
Dark Reader 4.9.26
Windows 10 fully updated

It's not something detrimental, just a little annoying. Hoping someone can help ^-^

**Comment by Gusted** [comment]
Hi @Oddiki!

Do you have set a custom font? 

Regards, 
Gusted

**Comment by Oddiki** [comment]
Hi @Gusted

Thanks for replying!

I can't remember changing any settings, so I tried to look at the text settings.
It's hard to see if it's changed since there is no default text, only different types. But at the bottom at that it is set to default and not custom, so I suppose it's at its default state. Unless that's only for the theme and not the text.

**Comment by Gusted** [comment]
Hi @Oddiki!

I meant if you changed the font settings, rather then the text settings. It is possible that a custom font doesn't have support for icons and shows such box.

Regards,
Gusted

**Comment by Oddiki** [comment]
@Gusted 

Sorry if I explained badly.

From what I'm seeing, font is under the tab 'More' under the dark reader settings. And the defualt / custom option is at the bttom at that tab. I'm guessing it's not correlated to the font though. 
But I can't tell if I have changed the font or not, since there is not default font. There are 7 fonts counting the one that is selected. It's set to Segoe UI, if this is not the defualt font, which font do I change it to?
